index.js

```diff
@@ -208,6 +215,7 @@ var BrowserStackBrowser = function (
           // https://github.com/browserstack/api/issues/10
           if (!sessionUrlShowed) {
             log.info('%s session at %s', browserName, worker.browser_url)
+            sessionMapping[self.id] = worker.browser_url.split('/').slice(-1)[0]
```

Comment:
this assumes that the session id is the last part of the URL - e.g. `https://www.browserstack.com/automate/builds/7c26d1b332278c0287293679c4d7c850fca2.../sessions/22e05303804be45a6464a38f7e06bb5510f00000`